### DISCUSSION TITLE
Backup system error pg_dump: error: aborting because of server version mismatch

### DISCUSSION THREAD

**Discussion by carlosecv** [Tecnativa/doodba#358] [Q&A] [answered]
Actually the backup system get this error for Odoo 14 and PostgreSQL 13

pg_dump: error: server version: 13.1; pg_dump version: 12.3
pg_dump: error: aborting because of server version mismatch

Somwonw has this issue with backups?

**Comment by yajo** [comment, marked as answer]
You must make sure the postgres client libraries version is >= than the postgres server version.

A quick fix for your situation will be to either produce the backup from the `db` container, or downgrade it to postgres 12, so its version is <= than the one in the odoo container.

To check what version you have in a given container, do:

```sh
container=odoo # It can be any service in the stack
docker-compose run --rm $container psql --version
```

**Reply by carlosecv** [reply]
Thank you

¿Its possible upgrade psql client on backup container to match version 13.0?

Thank you

**Reply by carlosecv** [reply]
docker-compose -f prod.yaml run --rm odoo psql --version
psql (PostgreSQL) 13.1 (Debian 13.1-1.pgdg90+1)

docker-compose -f prod.yaml run --rm backup psql --version
psql (PostgreSQL) 12.3

**Reply by joao-p-marques** [reply]
> ¿Its possible upgrade psql client on backup container to match version 13.0?

Hi! Do you have the latest version for the `backup` image? In the latest image we already use Postgres 13.
